index.html

```diff
@@ -1,16 +1,130 @@
 <!doctype html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+    <head>
+        <meta charset="utf-8">
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+        <title>Great Idea!</title>
 
-</head>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+        <link rel="stylesheet" href="css/index.css">
 
-<body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
-</body>
+    </head>
+
+    <body>  
+        <div class="container">
+            <nav class="nav">
+              <div class="nav-links">
+                <a href="#">Services</a>
+                <a href="#">Product</a>
+                <a href="#">Vision</a>
+                <a href="#">Features</a>
+                <a href="#">About</a>
+                <a href="#">Contact</a>
+              </div>
+                  <div class="main-logo">
+                  <img class="logo" src="https://github.com/tommaay/User-Interface/blob/master/great-idea-website/img/logo.png?raw=true" alt="Great Idea! Company logo.">
+                  </div>
+            </nav>
+        
+        
+            <header class="header group">
+              <section class="heading-wrapper">
+                <h1 class="heading">
+                  Innovation<br />
+                  On<br />
+                  Demand
+                </h1>
+        
+                <a class="btn" href="#">Get Started</a>
+              </section>
+              <div class="img-wrapper">
+                <img src="https://github.com/tommaay/User-Interface/blob/master/great-idea-website/img/header-img.png?raw=true" alt="Image of a code snippet.">
+              </div>
```

Comment:
you could select the image here by itself and avoid having to use a wrapper div for it. you will be able to do any of the manipulation that you did to the div to the img tag directly and avoid excessive divs